packages/core/src/temporal/ChronoField.js

```diff
@@ -120,58 +242,73 @@ export class ChronoField extends TemporalField {
     }
 
     /**
-     *
-     * @returns {string}
+     * @return {string}
      */
     name(){
         return this._name;
     }
 
     /**
-     *
-     * @returns {!number}
+     * @return {TemporalUnit} the period unit defining the base unit of the field.
      */
     baseUnit(){
         return this._baseUnit;
     }
 
     /**
-     *
-     * @returns {!number}
+     * @return {TemporalUnit} the period unit defining the range of the field.
      */
     rangeUnit(){
         return this._rangeUnit;
     }
 
     /**
-     *
-     * @returns {!ValueRange}
+     * @return {ValueRange} the range of valid values for the field.
      */
     range(){
         return this._range;
     }
 
     /**
-     *
      * @returns {string}
      */
     displayName(){
         return this.toString();
     }
 
     /**
+     * Checks that the specified value is valid for this field.
+     *
+     * This validates that the value is within the outer range of valid values
+     * returned by {@link range}.
+     *
+     * This method checks against the range of the field in the ISO-8601 calendar system.
      *
-     * @param {number} value
-     * @returns {*}
+     * @param {!number} value the value to check.
+     * @returns {number} the value that was passed in.
      */
     checkValidValue(value) {
         return this.range().checkValidValue(value, this.name());
```

Comment:
Looks like, i found an existing bug, should be `...checkValidValue(value, this)`. Doesn;t harm because the field is just used for printing

Reply:
I'll take a look into this.

Reply:
Well, fixed. As you said, since it's just for string concatenation there is no difference. Better to be consistent, nonetheless.

Reply:
Yes, it was definitely a bug